Modules/@babylonjs/react-native/shared/BabylonNative.cpp

```diff
@@ -1,6 +1,7 @@
 #include "BabylonNative.h"
 
 #include <Babylon/Graphics.h>
+#include <Babylon/GraphicsPlatform.h>
```

Comment:
After https://github.com/BabylonJS/BabylonNative/pull/656/files merges, we shouldn't need to include `GraphicsPlatform.h`

Reply:
Removed.